docs/manual-CN/2.query-language/3.language-structure/keywords-and-reserved-words.md

```diff
@@ -0,0 +1,173 @@
+# 关键字和保留字
+
+关键字是在 nGQL 中具有重要意义的单词。保留关键字需引用方可使用。
+
+非保留关键字无需引用可直接使用，且所有非保留字都会自动转换成小写，所以非保留字是不区分大小写。保留关键字需使用反引号标注方可使用，例如 \`AND\`。
+
+```ngql
+nebula> CREATE TAG TAG(name string);
+[ERROR (-7)]: SyntaxError: syntax error near `TAG'
+
+nebula> CREATE TAG SPACE(name string); -- SPACE 为非保留关键字
+Execution succeeded
+
+nebula> SHOW TAGS; -- 所有非保留字都会自动转换成小写
+=============
+| ID | Name |
+=============
+| 25 | space|
+-------------
+```
+
+`TAG` 为保留字使用时必须使用反引号。 `SPACE` 为非保留字使用时无需加反引号。
+
+```ngql
+nebula> CREATE TAG `TAG` (name string); -- 此处 TAG 为保留字
+Execution succeeded
+```
+
+## 保留字
+
+以下列表为 nGQL 中的保留字。
+
+```ngql
+ADD
+ALTER
+AND
+AS
+ASC
+BALANCE
+BIGINT
+BOOL
+BY
+CHANGE
+COMPACT
+CREATE
+DELETE
+DESC
+DESCRIBE
+DISTINCT
+DOUBLE
+DOWNLOAD
+DROP
+EDGE
+EDGES
+EXISTS
+FETCH
+FIND
+FLUSH
+FROM
+GET
+GO
+GRANT
+IF
+IN
+INDEX
+INDEXES
+INGEST
+INSERT
+INT
+INTERSECT
+IS
+LIMIT
+LOOKUP
+MATCH
+MINUS
+NO
+NOT
+NULL
+OF
+OFFSET
+ON
+OR
+ORDER
+OVER
+OVERWRITE
+PROP
+REBUILD
+RECOVER
+REMOVE
+RETURN
+REVERSELY
+REVOKE
+SET
+SHOW
+STEPS
+STOP
+STRING
+SUBMIT
+TAG
+TAGS
+TIMESTAMP
+TO
+UNION
+UPDATE
+UPSERT
+UPTO
+USE
+VERTEX
+WHEN
+WHERE
+WITH
+XOR
+YIELD
+```
+
+## 非保留关键字
```

Comment:
I think that it is enough to add reserved words here.
No need to add non-reserved words because it changes dynamically.
WDYT?

Reply:
it's here for reference.

Reply:
I do not think so.
Because non-reserved words and ordinary characters, there is no difference for users.
So the user knows that reserved words are enough.

Reply:
Although nGQL is case insensitive, we encourage to use the keywords in UPPERCASE, that's why I bother myself to add the non-reserved list. Thanks for your kind suggestion.